Library/Homebrew/test/spec_helper.rb

```diff
@@ -129,17 +128,12 @@
   # Never truncate output objects.
   RSpec::Support::ObjectFormatter.default_instance.max_formatted_output_length = nil
 
-  config.include(FileUtils)
```

Comment:
I suspect that this might be controversial. I wonder if we'd be better off just not including it anywhere but maybe that's going too far. I'm fine with it though.

Reply:
Yeah, I'm fine if folks feel otherwise and want to keep it here. I generally prefer loaders to be as minimal as is practical, but at the same time, `brew` itself is fundamentally a file util…

Reply:
Fine with these changes 👍🏻 ❤️